packages/react-examples/src/react/FocusTrapZone/e2e/FocusTrapZone.e2e.ts

```diff
@@ -1,24 +1,413 @@
+import type { IFocusTrapZoneProps } from '@fluentui/react/lib/FocusTrapZone';
+import type { FTZTestWindow } from './shared';
+
 const ftzStoriesTitle = 'Components/FocusTrapZone/e2e';
 
+/**
+ * Calls `window.setProps()` -- this must be defined by the story being tested
+ */
+function setProps(props: IFocusTrapZoneProps) {
+  cy.window().then(win => (win as FTZTestWindow).setProps!(props));
+}
+
 describe('FocusTrapZone', () => {
   before(() => {
     cy.visitStorybook({ qs: { e2e: '1' } });
   });
 
+  // These are basic tests of different props, but due to the reliance on focus behavior they're
+  // best done in the browser.
+  describe('Focus behavior based on default and explicit prop values', () => {
+    beforeEach(() => {
+      cy.loadStory(ftzStoriesTitle, 'PropValues');
+    });
+
+    it('Restores focus to FTZ when clicking outside FTZ', () => {
+      setProps({});
+
+      cy.get('#mid').focus();
+      cy.focused().should('have.id', 'mid');
+
+      // try to focus on button outside FTZ
+      cy.get('#before').focus();
+      // it focuses first button inside FTZ instead
+      cy.focused().should('have.id', 'first');
+    });
+
+    it('Does not restore focus to FTZ when clicking outside FTZ with isClickableOutsideFocusTrap', () => {
+      setProps({ isClickableOutsideFocusTrap: true });
+
+      // verify we can focus on button outside FTZ
+      cy.get('#before').focus();
+      cy.focused().should('have.id', 'before');
+    });
+
+    it('Focuses first element when focus enters FTZ does not have focus and first bumper receives focus', () => {
```

Comment:
```suggestion
    it('Focuses first element when focus enters FTZ, the first bumper should not receive focus', () => {
```
is this what you mean ?

Reply:
I'm guessing so? This is just copied from the previous test description. Some of the descriptions and comments seem to be a little too specific to the implementation (bumpers) instead of describing the behavior.